java/client/src/main/java/glide/api/models/commands/ZAddOptions.java

```diff
@@ -85,4 +88,30 @@ public String[] toArgs() {
 
         return optionArgs.toArray(new String[0]);
     }
+
+    /**
+     * Converts ZaddOptions into a GlideString[].
+     *
+     * @return GlideString[]
+     */
+    public GlideString[] toArgsBinary() {
```

Comment:
Function `public GlideString[] toArgsBinary()` could be implemented using this one-liner:

```java
public GlideString[] toArgsBinary() { return new ArgsBuilder().add(toArgs()).toArray(); }
```